src/Features/Core/Portable/AddImport/Remote/AbstractAddImportFeatureService_Remote.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.AddImport
     internal abstract partial class AbstractAddImportFeatureService<TSimpleNameSyntax>
     {
         private async Task<ImmutableArray<AddImportFixData>> GetFixesInRemoteProcessAsync(
-            RemoteHostClient.Session session, Document document, TextSpan span, 
+            SessionWithSolution session, Document document, TextSpan span, 
```

Comment:
Trying to understand.  Why is this a SessionWithSolution, and not just, say, a 'connection'?